### DISCUSSION TITLE
购买后好像没有升级成专业版，且无法对其他设备进行授权

### DISCUSSION THREAD

**Discussion by J-a-g** [getlantern/lantern-forum-cn#1047] [General]
你好管理员，9月份购买的专业版
PC上的lantern 标题没有显示lantern pro，不知道是否升级为专业版，电脑端好像已经可以翻墙，但是无法进去其他设备的授权
账号邮箱：13282221172@163.com （或者suchengjian2@gmail.com）有点忘记了，当时购买后也没有邮件通知。
 @麻烦看一下账号是否升级成专业版，为什么无法授权其他设备
![Upload![微信图片_20241124122627](https://github.com/user-attachments/assets/6250f817-6c80-4aa9-a08e-97a273e7ac56)
![微信图片_20241124122432](https://github.com/user-attachments/assets/8257216c-0045-4b22-9d5e-721026f92729)

**Comment by MaxMKMax** [comment]
@J-a-g 好的. 请您放心, 我已经帮您提交问题上去处理了. 谢谢您的耐心等待.

**Comment by greySkiesMarsh** [comment]
Hi there, I can see in our system that your account has been upgraded, and I see 3 Windows 10 devices have been authorized to your account.

Since you're having trouble logging in, I've cleared the devices in the system. Please use the following code to log back in using the email address you posted in this forum. It will add a complimentary month, and will get you connected to Lantern Pro. 

KJ33Q-GKMVV-B4GYV-PWBF8-4HV4P
